test/model.jl

```diff
@@ -367,6 +367,13 @@ function test_model()
         @test !JuMP.unset_silent(model)
         @test !MOI.get(backend(model), MOI.Silent())
     end
+
+    @testset "set_parameter" begin
+        mock = MOIU.UniversalFallback(JuMP._MOIModel{Float64}())
+        model = Model(with_optimizer(MOIU.MockOptimizer, mock))
+        @test JuMP.set_parameter(model, "aaa", "bbb") == "bbb"
+        @test MOI.get(backend(model), MOI.RawParameter("aaa")) == "bbb"
```

Comment:
Can you test that `MOI.get(model, ...)` also works ?

Reply:
It does not, I think it's because there's no `MOI.get(model::Model, attr::MOI.AbstractOptimizerAttribute)`. There's only for model, variable, and constraint.

If you want, I can add the following function:

```
"""
    get(model::Model, attr::MathOptInterface.AbstractOptimizerAttribute)

Return the value of the attribute `attr` from the model's MOI backend.
"""
function MOI.get(model::Model, attr::MOI.AbstractOptimizerAttribute)
    if MOI.is_set_by_optimize(attr) &&
       !(attr isa MOI.TerminationStatus) && # Before `optimize!` is called, the
       !(attr isa MOI.PrimalStatus) &&      # statuses are `OPTIMIZE_NOT_CALLED`
       !(attr isa MOI.DualStatus)           # and `NO_SOLUTION`
        _moi_get_result(backend(model), attr)
    else
        MOI.get(backend(model), attr)
    end
end
```

This seems to work.

Reply:
Done in f6a9164d66781d554527f6b1c97644e3323a70cd, please tell me if it's okay

Reply:
Added 4b2f34ca123bf88357bd6a06d2c4b1e6c53d620b because if we're doing this might as well add the tests for `Silent` too